ui/src/app/layout/topics/single-topic.component.html

```diff
@@ -450,3 +458,24 @@ <h4 class="modal-title">{{ 'Delete Schema' | translate }}</h4>
         </button>
     </div>
 </ng-template>
+
+<ng-template #updateTopicDescription let-dlg>
+    <div class="modal-header">
+        <h4 class="modal-title">{{ 'Edit' | translate }}</h4>
+        <button type="button" class="close" aria-label="Close" (click)="dlg.dismiss()">
+            <span aria-hidden="true">&times;</span>
+        </button>
+    </div>
+    <div class="modal-body">
+        <div class="mt-3">
+            <textarea class="form-control" rows="5" cols="11" [(ngModel)]="updatedTopicDescription"></textarea></div>
```

Comment:
Where is `updatedTopicDescription` initialized? Looks to me as if the textarea is empty, even if the topic already had a description before... (only from looking at the code, so maybe I am missing something)

Reply:
done.